config/mirrorbits.yaml

```diff
@@ -75,11 +72,10 @@ repository:
         - nobrl
         - serverino
         - cache=strict
-
 image:
   mirrorbits:
     tag: v0.1.2
     pullPolicy: IfNotPresent
   files:
-    tag: 2.4
+    tag: 2e064dc3197af3fa33704936040165b34a3c03b55024d90ea11b2f9d5bcde552
```

Comment:
We have to specify that it's a digest if I'm not mistaken (or apply the same kind of advanced process for `httpd` like we did with `nginx`)